javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
+    return this.head === null;
   }
 
+  // Add a Node to the front of a linked list
   insert(value) {
-    //insert a node at the beginning of a given LL
-
-    if (this.isEmpty() === true) {
-      // the list is empty
-      // make a new Node with the value given
-      const node = new Node(value);
```

Comment:
Since you're doing this in both cases, why not pull it up to the top and do it once. You'll often hear developers talk about DRY code which stands for "don't repeat yourself." My team usually just leaves comments with ☔  or 💧 on PRs :).

Note this also applies to `this.head = node` later in the function.